sdcm/utils/health_checker.py

```diff
@@ -16,7 +16,13 @@ def check_nodes_status(nodes_status, current_node):
     for node_ip, node_properties in nodes_status.items():
         if node_properties['status'] != "UN":
             is_target = current_node.print_node_running_nemesis(node_ip)
-            ClusterHealthValidatorEvent(type='critical', name='NodeStatus', status=Severity.CRITICAL,
+            LOGGER.info(f'REMOVED NODES LIST = {[node.ip_address for node in removed_nodes_list]}')
+            if node_ip in [node.ip_address for node in removed_nodes_list]:
```

Comment:
First thing: 
this won't work since `ip_address` property refreshes the instance state by running `self._refresh_instance_state()` (at least for GCE) on the dead node, it will fail... Currently, we don't support keeping dead Node objects (many properties can't work since they assume instance that is alive). 

Second thing:
from an architecture point of view it is better to keep the list of dead nodes in the Cluster object, it can be useful for future features and you don't need to pass it everywhere and add to the list in all places the you call terminate node. So when removing a node (`sdcm.cluster.BaseCluster.terminate_node` ), add it to the list of the removed nodes.